lib/Env.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ExponentPhpSDK;
+
+class Env {
```

Comment:
I think we don't really need this class and we can just rely on the dotenv package right?